notebook.ipynb

```diff
@@ -139,9 +141,9 @@
     {
      "data": {
       "text/plain": [
-       "{'hidden_states': array([[-0.00722676,  0.00624354, -0.0079936 , ...,  0.00659193,\n",
-       "          0.00973766,  0.0026458 ]], dtype=float32),\n",
-       " 'logits': array([[0.6862614, 0.6639196, 0.5106052]], dtype=float32)}"
+       "{'hidden_states': array([[-0.00709132,  0.00617745, -0.00790404, ...,  0.00672843,\n",
```

Comment:
Maybe we should use a fixed seed to avoid changing the output for the random signal in every PR.

Reply:
Yes, that would be a good idea.

Reply:
I created https://github.com/audeering/w2v2-how-to/issues/4